arch/arm/boot/dts/overlays/rpi-poe-overlay.dts

```diff
@@ -89,6 +79,52 @@
 		};
 	};
 
+	fragment@3 {
+		target = <&firmware>;
+		__overlay__ {
+			poe_pwm: pwm {
+				compatible = "raspberrypi,firmware-poe-pwm";
+				#pwm-cells = <2>;
+			};
+		};
+	};
+
+	fragment@4 {
+		target = <&i2c0>;
+		__dormant__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+
+			poe@0x51 {
```

Comment:
The preferred name is `poe@51`.

Reply:
Not sure what I was copying that used the 0x. Ho hum.